packages/expo-modules-core/ExpoModulesCore.podspec

```diff
@@ -114,6 +114,9 @@ Pod::Spec.new do |s|
 
   s.test_spec 'Tests' do |test_spec|
     test_spec.dependency 'ExpoModulesTestCore'
+    if reactNativeTargetVersion >= 77
+      test_spec.dependency 'ReactAppDependencyProvider'
```

Comment:
Can we move this dependency to the `expo` package? Adding it here will increase the building time - see https://github.com/expo/expo/pull/34985.

Reply:
done